extension/src/project/openWebHelp.ts

```diff
@@ -0,0 +1,174 @@
+import * as vscode from 'vscode';
+import { webHelpContainer } from "../completion/autocompletionProvider"; // Currently where all ./data/ files are being loaded & was maintained for symmetry
+
+
+// Triggers VSCode to open the official web documentation URL related to the currently selected symbol
+// Note: this function is directly referenced by projectCommands.ts and the the package.json contributes (commands & menus) section.
+export function openWebHelp() {
+	const editor: vscode.TextEditor = vscode.window.activeTextEditor;
+	let selected: string = editor.document.getText(editor.selection);	
+	let urlPath: string = webHelpContainer.getWebHelpUrlBySymbolName(selected);
+	vscode.env.openExternal(vscode.Uri.parse(urlPath));
+}
```

Comment:
```suggestion
export async function openWebHelp() {
	const editor: vscode.TextEditor = vscode.window.activeTextEditor;
	let selected: string = editor.document.getText(editor.selection);
	if (selected === "") {
		await vscode.commands.executeCommand('editor.action.addSelectionToNextFindMatch');
		selected = editor.document.getText(editor.selection);
	}
	let urlPath: string = webHelpContainer.getWebHelpUrlBySymbolName(selected);
	vscode.env.openExternal(vscode.Uri.parse(urlPath));
}
```
This will allow right clicking on a word without having to select it.

Reply:
I could combine this with a regex to identify the nearest contiguous string, but snippet you added doesn't actually provide a selection from an empty string starting point. However at that point I really wouldn't need to execute addSelectionToNextFindMatch anyway.

Reply:
My goal was to allow the user to right click on a word choose help without having to select it. The 2 lines I added simply select the word under the cursor.

Reply:
I had a big long reply about how _addSelectionToNextFindMatch_ wasn't properly awaiting and that the text wasn't being selected until after the command finished, but decided to give it another try first and for some reason its working now.... Maybe I forgot to run the bat file or something; IDK, but its working!

I will say this is some interesting tribal knowledge because _addSelectionToNextFindMatch_ implies that you already have a selection.